cypher-shell/src/main/java/org/neo4j/shell/DatabaseManager.java

```diff
@@ -7,6 +7,7 @@
  */
 public interface DatabaseManager
 {
+    String ABSENT_DB_NAME = "";
```

Comment:
Driver team wants us to not set the database parameter instead of setting it to an empty string. We can do that in the next refinement round, since we should now also be able to get the actual database name from the result meta data, which should feed into the prompt instead of hard coding it to the DEFAULT_DEFAULT_DB_NAME when we don't know.

Reply:
Okay, leaving this for later.